tests/unit_test.py

```diff
@@ -170,7 +170,9 @@ def suggested_artifact(*args, **kwargs):
     reason="windows Auto-installing native modules is not supported "
            "when python >= 3.10")
 def test_autoinstall_protobuf(reuse):
-    kws = {"package_name":"protobuf", "module_name":"google.protobuf"}
```

Comment:
I dont know how you want to handle this, but I just added the kwargs to _use_str and added a comment that they're for internal use.

Reply:
okay.